### PR TITLE
Carousel - Empty Slot

### PR DESCRIPTION
**Pull request by RodTGG**
Small feature request as posted in the following

[Discussion: 1455](https://github.com/orgs/primefaces/discussions/1455)

Added an Empty slot so that the component behaves similar to other data components. e,g, DataView, DataTable

### CODE REVIEW

**Comment by tugcekucukoglu** [comment]
Could you create an issue?

**Comment by RodTGG** [comment]
> Could you create an issue?

Can do, i was told to create a discussion first for feature requests

File: components/lib/carousel/Carousel.vue [hunk 1]
```diff
@@ -4,90 +4,94 @@
...
+                            :tabindex="d_page === i ? '0' : '-1'"
+                            :aria-label="ariaPageLabel(i + 1)"
+                            :aria-current="d_page === i ? 'page' : undefined"
+                            @click="onIndicatorClick($event, i)"
+                            v-bind="ptm('indicatorButton', getIndicatorPTOptions(i))"
+                        />
+                    </li>
+                </ul>
+            </template>
+            <div v-else>
```
Comment (tugcekucukoglu) [inline components/lib/carousel/Carousel.vue:93]: You wrote in the documentation that an empty slot was added, but you did not make any changes about it.

**Comment by tugcekucukoglu** [comment]
This PR adds a new feature rather than solving a bug. Enhancement issue report would be more appropriate.

**Review by tugcekucukoglu — APPROVED** [review]

**Pull request merged**